File: Modul1/Python/PRAK104-2310817310012-MUHAMMAD_AZWIN_HAKIM.py
a = 400000
b = 350000
da =  13 / 100
db =  21 / 100
hasil_a = a - (a * da)
hasil_b = b - (b * db)

print(f"Harga sepatu A adalah {a}")
print(f"Harga sepatu B adalah {b}")
print(f"Sepatu A mendapat diskon {int(da * 100)}% sehingga harganya menjadi {int(hasil_a)}")
print(f"Sepatu B mendapat diskon {int(db * 100)}% sehingga harganya menjadi {int(hasil_b)}")

File: Modul1/C/PRAK104-2310817310012-MUHAMMAD_AZWIN_HAKIM.c
#include <stdio.h>
int main(){
    int a = 400000;
    int b = 350000;
    double da = (double) 13/100;
    double db = (double) 21/100;
    double hasil_a = a - (a * da);
    double hasil_b = b - (b * db);
    printf("Harga sepatu A adalah %d\n", a);
    printf("Harga sepatu B adalah %d\n", b);
    printf("Sepatu A mendapat diskon %.0f%% sehingga harganya menjadi %.0f\n", da * 100, hasil_a);
    printf("Sepatu B mendapat diskon %.0f%% sehingga harganya menjadi %.0f", db * 100, hasil_b);
    return 0;
}